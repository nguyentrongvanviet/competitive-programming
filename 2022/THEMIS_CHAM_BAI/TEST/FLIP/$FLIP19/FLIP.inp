102 873
835521 244448
506059 377951
760477 571756
477222 634977
789159 353057
877970 700567
404949 236548
175191 683194
396454 806742
538448 175649
654298 945720
281528 261272
515020 339495
661662 703446
617737 404771
848888 806841
54912 221016
193211 565759
716402 517629
235133 390862
778282 723164
889783 347400
117578 809891
946634 821720
407727 555350
770382 164030
85917 443667
65247 90441
486469 678523
837883 347300
603602 660341
606130 374678
270926 312025
917105 855315
409603 565844
270927 907896
935106 431040
945469 534169
767135 583778
440141 523925
139328 607910
527640 788216
538613 340484
584795 749061
253195 325763
537826 506100
816107 716431
258020 321868
50701 338538
916148 306685
904060 691105
359036 115139
412423 81958
390087 979693
188973 59753
280396 576524
208216 150846
222191 52557
397647 557486
984410 648921
449707 518130
221509 816585
825875 902882
368719 236195
242064 438966
349553 686220
187261 88520
33685 982109
703929 697088
591312 70174
590878 913346
928523 112564
144416 496151
583691 732498
252831 96104
779485 404853
654044 328835
296967 228695
69240 610537
599545 517622
503897 909820
440243 511842
5618 23749
633919 12872
952409 574121
543583 151297
152684 144992
358387 476985
757123 782655
30345 588424
49236 207410
682322 642640
57123 451380
632411 755071
907865 168360
654221 496145
781356 689117
846675 516986
760703 805359
53692 156156
9164 123562
545320 519120
279625
389538
280028
827042
117367
626948
732427
610476
939298
177009
629313
892086
906525
299844
731030
399093
222802
840113
132465
608957
117030
776741
263081
328180
691810
391809
135991
353151
394868
89381
249679
313100
796407
893163
596738
15710
414293
673017
978638
725690
750240
209988
791474
995224
767281
921278
903987
7572
1883
556176
526757
821707
904100
327909
702858
389731
511091
542238
357456
549410
170801
745197
687409
516073
56077
344209
68818
670426
152193
113501
961210
134319
857092
831145
144678
460504
334650
285491
53191
547637
539883
336024
139612
872234
931638
41903
724347
788737
187715
140088
794610
179788
548695
668269
500700
327671
737635
558212
453058
738214
791824
190486
664379
846162
920233
797424
949474
246771
832716
854401
62738
956343
399752
766937
603463
745814
483440
584864
671109
652401
764121
705325
958737
674807
540777
968300
538422
812385
787238
695246
680083
288294
616378
791032
960746
340821
535242
238200
931580
602397
983655
284688
698837
362702
906308
96534
615803
129541
32903
58606
915755
558781
349686
532480
226087
422658
429238
195269
744370
647648
330158
172451
572884
223462
474291
11373
612540
520911
564935
50871
55242
819784
259412
343590
852070
345118
911412
719897
931794
392622
787596
726045
972300
631603
587818
767229
560735
279303
687118
959477
489976
495580
154127
816968
337232
917829
220137
468549
719218
180873
674914
774506
705910
938674
437483
658529
735930
791097
558728
248140
931019
129211
577368
879962
356035
66698
239182
72131
972747
686250
611365
723174
401327
489721
767184
903655
627838
571616
845728
853680
413313
6418
122408
697507
217812
990517
705034
664453
752208
924347
954068
225116
440372
514848
510344
929019
343579
965350
961630
883136
309751
33940
979163
959263
504155
143835
203489
829373
898518
550597
598115
462444
418764
515659
717906
268957
529594
202243
76416
828622
563529
727296
303440
932889
427904
934730
160743
932680
941869
186641
291877
931048
861916
83822
308112
809635
503695
42319
523977
527353
791963
777355
271452
234704
688640
938353
682879
665853
34091
202294
112834
951751
490645
493286
680673
460304
594200
264846
766040
751516
638871
476815
871885
112481
836619
198904
690049
411727
725956
181794
454375
942919
668214
576024
306786
737320
613042
836346
986604
561523
847815
420593
689682
15536
633623
604013
219429
793057
210357
940424
909327
452386
410284
119674
83397
791812
333012
5725
198225
103027
500211
18579
306983
801128
796880
454152
350973
829140
623195
616520
733078
454786
350174
247074
247262
887130
817967
984517
660882
94283
706271
754583
322985
481270
9957
73416
168045
718782
584592
582073
99343
630732
766389
673636
530238
17915
787426
423889
433953
909282
11386
814119
101973
991322
21906
713964
222051
268337
834052
750138
146706
238647
325604
287727
346896
704253
918045
927586
118549
440634
628779
570733
1920
75528
520865
949006
58358
51044
226652
424359
296492
953332
921238
441625
287506
43902
603945
994032
984523
175087
483675
676744
757327
130548
769113
800808
722067
864002
967560
152437
456864
148150
766041
718377
465758
41561
1049
503589
75981
271308
508776
659171
430023
182535
419384
636809
880281
739038
776800
968511
418073
43250
318346
84061
647823
691373
741889
127688
666896
152571
78012
356296
747951
183393
89737
117439
435186
776840
963275
934325
356655
773464
130132
141010
238821
971160
894158
5843
651547
846636
712903
682761
220039
969599
489628
57471
25667
399170
990043
336684
442533
324836
704789
154600
940303
297508
749037
186863
753157
612172
356516
537276
560624
126392
753812
16713
8121
843312
702665
592800
795744
447323
221928
208465
798775
656111
185608
843970
640889
76404
44580
463209
46980
199732
158288
684056
308734
990962
146317
757023
124174
187887
115600
420826
195589
910056
290719
870653
725537
786901
241049
681051
952765
833226
119988
679098
726246
728460
562549
359670
270479
227569
928030
404306
376006
471340
367459
886889
470890
696018
638542
115375
849692
269241
115684
53778
885606
101220
295568
267387
71597
791139
220518
956601
335659
943767
943925
672136
587290
298661
641963
20751
8608
798972
352629
264059
844431
970018
453487
4872
685658
934456
86444
336743
316621
702101
279414
621753
310231
731980
912587
798317
642590
110776
385283
78324
206116
521171
832392
603279
546202
806262
438857
887618
404758
570389
257023
270365
468726
703216
935256
894006
61593
333334
208570
401985
621881
564751
71394
501395
271689
445478
670738
131128
340943
626906
417354
135414
575826
997568
801861
311932
397834
705634
304616
614709
266511
409734
585599
263261
619859
138845
842701
919541
684331
397365
175190
509827
10506
34201
913346
756429
281790
330493
647774
16941
461757
741035
575544
487275
281273
594447
967439
847847
539179
983689
990251
926541
917858
774004
227341
355014
216186
1066
856958
360824
548413
743683
180080
751025
801373
933166
677102
631158
198424
741293
333730
916794
902433
340180
987288
97865
363807
424738
661048
756727
625124
977903
772920
947714
426062
582240
651734
807665
741062
489443
504066
622552
122380
717149
610980
647142
60010
761799
228098
303432
582481
726047
506379
198739
961507
339570
750795
259971
291396
738396
364699
574511
743877
237534
90903
592794
338299
566357
645179
524466
660506
230030
768744
75909
144457
484638
477904
938096
32478
386552
604229
859343
813866
796233
111581
278904
294003
758078
868852
206686
882216
238971
592044
373631
499372
184648
333596
186789
326794
875495
560121
649116
798040
974734
320710
778863
15743
895081
847002
860284
717173
593619
291600
804031
607478
733778
794129
107718
386856
490861
344576
587197
609849
140517
311442
589840
686998
558193
73463
551709
583652
574116
338367
965274
421905
778824
290343
808230
97395
61351
569720
381995
151504
961423
556046
60975
902055
534953
741354
990569
41338
618213
643150
901580
241463
923834
129333
527103
991236
14551
277136
69813
333656
92671
628006
982393
442082
429313
409026
913928
27829
278577
148992
26562
756334
49787
278359
756279
843247
500408
138925
467302
406316
101868
636070
973244
463229
