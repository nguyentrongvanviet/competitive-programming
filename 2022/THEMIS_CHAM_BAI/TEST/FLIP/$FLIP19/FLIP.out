835521 506059 571756 634977 789159 700567 404949 683194 806742 538448 654298 281528 515020 703446 617737 806841 54912 565759 517629 390862 778282 889783 809891 821720 555350 770382 85917 90441 486469 837883 603602 606130 312025 855315 565844 907896 935106 534169 583778 523925 607910 527640 538613 584795 325763 506100 816107 321868 50701 916148 691105 115139 81958 979693 59753 576524 208216 52557 557486 984410 518130 816585 825875 368719 242064 686220 88520 982109 703929 591312 590878 928523 496151 583691 96104 779485 654044 296967 610537 517622 503897 511842 5618 633919 574121 543583 152684 476985 757123 588424 49236 642640 57123 632411 907865 496145 781356 516986 760703 53692 9164 519120 